pysrc/pipfile.py

```diff
@@ -3,7 +3,7 @@
 This only extracts a small subset of the information present in a Pipfile,
 as needed for the purposes of this library.
 """
-from utils import is_string
+import utils
```

Comment:
<img src="https://media3.giphy.com/media/nbvFVPiEiJH6JOGIok/giphy.gif"/>

https://www.datacamp.com/blog/lessons-from-the-zen-of-python

> Namespaces are one honking great idea -- let's do more of those!